### PR TITLE
Add minutes from Community Call on 2022-12-08

### PR DESCRIPTION
**Pull request by itrich**
Signed-off-by: Eduard Itrich <eduard@itrich.net>

### CODE REVIEW

**Comment by itrich** [comment]
I'll proof-read this once again and then open the PR for reviews

File: community-call/20221208.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+## Team Updates
+
+### Team IaaS (Sprint review and planning)
+* Test setup with multiple Keycloaks
+* Validation limited to ceph clusters deployed by OSISM
+* image-manager documentation proceeding
+* image-crawler can crawl three image at the moment. Needs to be integrated with image-manager
+* SOS-report functionality on hold at the moment
+* Benchmarks to determine overhead of disk encryption in ceph
+* Wiping data from local disks needs work (currently lvremove is insufficient!)
```
Comment (garloff) [inline community-call/20221208.md:25]: better: (currently used lvremove is insufficient)

File: community-call/20221208.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+* Cloud-in-a-box node image works well
+    
+Minutes: https://github.com/SovereignCloudStack/minutes/blob/main/iaas/20221207.md
+
+### Team Container (Sprint review and planning)
+* Local storage ADR was merged
+* Test cases for loadbalancer beahvior
+* Tests should be added to a test collection
```
Comment (garloff) [inline community-call/20221208.md:33]: This and the next bullet should have an additional indentation (they are sub items to the loadbalancer behavior). Also beahvior -> behavior.

File: community-call/20221208.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+* Next weeks: Additional storage classes discussion on 2022-12-19
+
+Minutes: https://github.com/SovereignCloudStack/minutes/blob/main/container/20221205.md
+
+**No team meeting on 2022-12-12!**
+
+### Team Ops, IAM (Sprint review and planning)
+
+* Review of work happening on device-authorization grant by @reqa
+* PoC works, yeah!
```
Comment (garloff) [inline community-call/20221208.md:47]: This and the next one: Additional indentation (they are subitems to device auth grant)

File: community-call/20221208.md [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+### SIG Standardization
+@alexander-diab presented his current understanding of a certification program and raised several questions.
+* In a first step, we'll have a "self-service" certification program that can be completed via a given toolsuite
+* Discussion about versioning/life-cycle of certification levels. 
+* Closer look at https://github.com/SovereignCloudStack/Docs/pull/148
+
+Minutes: not present yet
+
+### Overflow Slot (k8s on the management plane)
+Discussion on deploying a k8s cluster on top of the management plane that can host additional applications/services. Everybody agreed that this is a good idea. Additional discussion on how this can be achieved/implemented (e.g. via k3s, minio, etc.)
```
Comment (garloff) [inline community-call/20221208.md:81]: s/minio/minikube/

File: community-call/20221208.md [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+
+## Standardization
+* SCS-0001 stabilized on 2022-11-28
+* Please review https://github.com/SovereignCloudStack/Docs/pull/148
+* Several open issues to write decision records. Please contribute to these tasks! See https://github.com/SovereignCloudStack/issues
+
+## Community
+* SCS Hackathon #2 planned for early March 2023
+* gx-scs has been cleaned up and will be hoepfully be available again next week.
```
Comment (garloff) [inline community-call/20221208.md:90]: s/hoep/hope/

**Review by garloff — CHANGES_REQUESTED** [review]
Please perform the minor corrections! TIA!

**Comment by fkr** [comment]
I actually wonder why we duplicate the contents of the minutes of each team call. Imho it should suffice to point to the minutes from that call for the technical stuff that is being reported.
Why am I suggesting that: Because I want people to actually read the original minutes of the team call and to me this seems duplicate effort with the danger of actually not being as complete as the team call minutes.

Furthermore we agreed a while back (at least @berendt and I suggested that) that we don't do the participants tracking for the community call.

**Comment by itrich** [comment]
> I actually wonder why we duplicate the contents of the minutes of each team call. Imho it should suffice to point to the minutes from that call for the technical stuff that is being reported. Why am I suggesting that: Because I want people to actually read the original minutes of the team call and to me this seems duplicate effort with the danger of actually not being as complete as the team call minutes.

I followed the minutes recorded during earlier calls without any further intention. Just providing the link to the corresponding minutes will drastically reduce the  difficulty to take minutes. On the other hand, providing a short summary of the most important hightlights offers readers the opportunity to grasp what is happening without having to dig through secondary documents. (Which is what we should strive for IMHO). I'd propose to keep the summary if the minute taker is willing to record one.

> Furthermore we agreed a while back (at least @berendt and I suggested that) that we don't do the participants tracking for the community call.

You're right. I removed the participants' list from the minutes.

**Comment by itrich** [comment]
@garloff I processed your comments and IMHO this PR can be reviewed again.

**Review by garloff — APPROVED** [review]

**Pull request merged**